Draft a project idea for `GSoD` (#118)
keptn/community: mentorship/gsod/2022/project-ideas.md

# GSoD Project Ideas - Keptn

This document presents Google Season of Docs project ideas
which Keptn contributors have proposed for this year.
Application guidelines and more information about Keptn
in GSoD are avaialble on [this page](../README.md)

## 2022 Project Ideas

- [Documentation assessment](#keptn-docs-assessment) 

<a name="keptn-docs-assessment"></a>
### Documentation assessment

> This project is to have a docs assessment performed, to help us come with a plan for improving our documentaion. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end users.

Reviewer: ```suggestion
This project would assess the Keptn documentation, to help us come with a plan for improving our documentation. In particular, we're looking for someone to look at the content organization, the clarity of the language and concepts, and to make sure it's as readable as possible for both implementors and end users.
```

- More info: [tech docs](https://github.com/cncf/techdocs/blob/main/assessments/howto.md) and [criteria](https://github.com/cncf/techdocs/blob/main/assessments/criteria.md)
- Slack channel for technical discussion: `#keptn-docs`
- Areas to study/improve: Technical writing, Documentation, English, Git
- Project size: 
- Difficulty: Easy

> - Potential mentor(s): 

Reviewer: ```suggestion
- Potential mentor(s): Meg McRoberts
```
